TwitchDownloaderCore/TwitchHelper.cs

```diff
@@ -162,39 +164,67 @@ public static async Task<EmoteResponse> GetThirdPartyEmoteData(string streamerId
                     string name = emote["code"].ToString();
                     string mime = emote["imageType"].ToString();
                     string url = String.Format("https://cdn.betterttv.net/frankerfacez_emote/{0}/[scale]", id);
-                    data.FFZ.Add(new EmoteResponseItem() { Id = id, Code = name, ImageType = mime, ImageUrl = url });
+                    emoteReponse.FFZ.Add(new EmoteResponseItem() { Id = id, Code = name, ImageType = mime, ImageUrl = url });
                 }
             }
 
             if (getStv)
             {
-                JArray STV = JArray.Parse(await httpClient.GetStringAsync("https://7tv.io/v2/emotes/global"));
+                string globalEmoteResponse = await httpClient.GetStringAsync("https://7tv.io/v3/emote-sets/global");
+                JsonElement globalEmoteElement = JsonDocument.Parse(globalEmoteResponse).RootElement.GetProperty("emotes");
+                JArray stvEmotes = JArray.Parse(globalEmoteElement.ToString());
 
                 if (streamerId != null)
                 {
                     //Channel might not have 7TV emotes
                     try
                     {
-                        STV.Merge(JArray.Parse(await httpClient.GetStringAsync(String.Format("https://api.7tv.app/v2/users/{0}/emotes", streamerId))));
+                        string streamerEmoteResponse = await httpClient.GetStringAsync(string.Format("https://7tv.io/v3/users/twitch/{0}", streamerId));
+                        JsonElement streamerEmoteSetElement = JsonDocument.Parse(streamerEmoteResponse).RootElement.GetProperty("emote_set");
```

Comment:
I would just use JObject here if you can (this JsonElement is a different json library I believe? might be wrong).
Don't know if this compiles, but I would try this:
```
JObject stvChannel = JObject.Parse(await httpClient.GetStringAsync("....." + streamerId));
stvEmotes.Merge((JArray)stvChannel["emote_set"]["emotes"]);
```

Reply:
Yep that works. Also using that syntax to simplify the global response.